<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thresholdSetting</class>
 <widget class="QDialog" name="thresholdSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>697</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>阈值设置</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>660</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>92</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>491</width>
     <height>421</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>491</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>491</width>
     <height>500</height>
    </size>
   </property>
   <property name="title">
    <string>Ａ面待检区</string>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>JALVC164245</string>
    </property>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>GH137S</string>
    </property>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_30">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A6"/>
     <widget class="QLabel" name="label_31">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A7"/>
     <widget class="QLabel" name="label_32">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A8"/>
     <widget class="QLabel" name="label_33">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A9"/>
     <widget class="QLabel" name="label_34">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A10"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>JDSPF28335</string>
    </property>
    <widget class="QSplitter" name="splitter_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_35">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A19"/>
     <widget class="QLabel" name="label_36">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A20"/>
     <widget class="QLabel" name="label_37">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>13</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A21"/>
     <widget class="QLabel" name="label_38">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>14</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A22"/>
    </widget>
    <widget class="QSplitter" name="splitter_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_39">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>15</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A23"/>
     <widget class="QLabel" name="label_45">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>16</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A24"/>
     <widget class="QLabel" name="label_46">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>17</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A25"/>
     <widget class="QLabel" name="label_47">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>18</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_A26"/>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>SM1040</string>
       </property>
       <widget class="QSplitter" name="splitter_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_48">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>19</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A11"/>
        <widget class="QLabel" name="label_49">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>20</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A12"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string>J16H281</string>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_50">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>21</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A13"/>
        <widget class="QLabel" name="label_51">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>22</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A14"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_6">
       <property name="title">
        <string>SM490</string>
       </property>
       <widget class="QSplitter" name="splitter_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_52">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>23</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A15"/>
        <widget class="QLabel" name="label_53">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>24</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A16"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7">
       <property name="title">
        <string>100</string>
       </property>
       <widget class="QSplitter" name="splitter_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_54">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>25</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A17"/>
        <widget class="QLabel" name="label_55">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>26</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_A18"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_13">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>420</y>
     <width>491</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>491</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>491</width>
     <height>241</height>
    </size>
   </property>
   <property name="title">
    <string>B面待检区</string>
   </property>
   <widget class="QGroupBox" name="groupBox_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>GH137S</string>
    </property>
    <widget class="QSplitter" name="splitter_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>451</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B1"/>
     <widget class="QLabel" name="label_15">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B2"/>
     <widget class="QLabel" name="label_16">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B3"/>
     <widget class="QLabel" name="label_17">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B4"/>
     <widget class="QLabel" name="label_18">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_B5"/>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QGroupBox" name="groupBox_16">
       <property name="title">
        <string>JT6H281</string>
       </property>
       <widget class="QSplitter" name="splitter_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B6"/>
        <widget class="QLabel" name="label_20">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B7"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_17">
       <property name="title">
        <string>SM1040</string>
       </property>
       <widget class="QSplitter" name="splitter_9">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_21">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B8"/>
        <widget class="QLabel" name="label_22">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B9"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QGroupBox" name="groupBox_18">
       <property name="title">
        <string>121</string>
       </property>
       <widget class="QSplitter" name="splitter_10">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_23">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B10"/>
        <widget class="QLabel" name="label_24">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>11</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B11"/>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_19">
       <property name="title">
        <string>100</string>
       </property>
       <widget class="QSplitter" name="splitter_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>211</width>
          <height>31</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QLabel" name="label_25">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>12</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B12"/>
        <widget class="QLabel" name="label_26">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>13</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_B13"/>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
